packages/plugin-core/src/commands/GoToSiblingCommand.ts

```diff
@@ -150,37 +151,57 @@ export class GoToSiblingCommand extends BasicCommand<
     return { data: { sibling } };
   }
 
-  private getSiblingsForJournalNote = (
-    notes: NotePropsByIdDict,
+  private getSiblingsForJournalNote = async (
+    engine: ReducedDEngine,
     currNote: NoteProps
-  ): NoteProps[] => {
-    const monthNote = notes[currNote.parent!];
-    const yearNote = notes[monthNote.parent!];
-    const parentNote = notes[yearNote.parent!];
+  ): Promise<NotePropsMeta[]> => {
+    const monthNote = await engine.getNoteMeta(currNote.parent!);
```

Comment:
nit: instead of using '!', we can first check if(!currNote.parent) then return []. This also saves 1 call (potentially over the network if it's a remote backend.)